app/models/customer.rb

```diff
@@ -171,6 +171,18 @@ def applicable_invoice_grace_period
     billing_entity.invoice_grace_period || 0
   end
 
+  def applicable_subscription_invoice_issuing_date_anchor
+    return subscription_invoice_issuing_date_anchor if subscription_invoice_issuing_date_anchor.present?
+
+    billing_entity.subscription_invoice_issuing_date_anchor
```

Comment:
More of a personal preference but since we can't have a `false` value for `subscription_invoice_issuing_date_anchor`, this could be written:

```suggestion
    subscription_invoice_issuing_date_anchor || billing_entity.subscription_invoice_issuing_date_anchor
```